float32 timestamp
uint32 index
float32 confidence
float32[] norm_pos
